consul/libtofs.jinja

```diff
@@ -0,0 +1,115 @@
+{%- macro files_switch(
```

Comment:
This macro is unused for this particular PR, so please remove this file for now (and add it in #58).  Also, `docs/TOFS_pattern.rst` should also be added in #58 as well.